### PR TITLE
Li component moved to a separate file

### LINKED ISSUE #6013: Move the li component to a separate file
### What would you like to share?

We can see an inline `li` component on line 11 of the component `components/mdx/ComponentStyles.js`

image

## Changes Requested

Please move this inline Component to a separate file `components/mdx/Li.js` and import it in the `ComponentStyles.js` component.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by pratham-yuxo**
## Fixes Issue #6013 

fixes #6013 

## Changes proposed

earlier the code was like this
231467703-72c79f43-b767-4716-a890-051f2756467f

Now , I have made a separate file for Li component

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

If there are any suggestions and some other good way to do this, then you can tell me.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6023"></a>

### CODE REVIEW

File: components/mdx/Li.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default function Li(props){
+    return (
+        <li className="w-full break-all" {...props}></li>
+    )
+}
```
Comment (eddiejaoude) [inline components/mdx/Li.js:5]: I don't think this is following our coding style

```suggestion
export default function Li(props) {
  return <li className="w-full break-all" {...props}></li>;
}
```

File: components/mdx/ComponentStyle.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import Li  from "./Li";
```
Comment (eddiejaoude) [inline components/mdx/ComponentStyle.js:1]: This is correct but doesn't follow the same style as the other imports

```suggestion
import Li  from "@components/mdx/Li";
```

File: components/mdx/Li.js (hunk 1 above)
Comment (pratham-yuxo) [reply components/mdx/Li.js]: Thank you for your suggestion. I'll try to do better next time.

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**